dashboard/test/factories.rb

```diff
@@ -530,8 +530,7 @@
   factory :survey_result do
     user {create :teacher}
     kind 'Diversity2016'
-    properties {{survey2016_ethnicity_asian: "1"}}
-    properties {{survey2016_foodstamps: "3"}}
+    properties {{ethnicity_asian: "1", foodstamps: "3"}}
```

Comment:
Hmm. My [data fix](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/oneoff/data_fix/survey_results_remove_year) removed the "survey" also. Let's discuss quickly offline.